api/v1/models/UniversalTrackingItem.js

```diff
@@ -0,0 +1,17 @@
+var Model = require('./Model');
+
+var UniversalTrackingItem = Model.extend({
+    tableName: 'event_tracking',
+    idAttribute: 'id',
+    hasTimestamps: ['created'],
+    validations: {
```

Comment:
Right. So if I hit the tracking endpoint between those times, that is the event that will be triggered?
Also - is there anyway we can modify it so we can pass a start and end time OR duration?